### ISSUE TITLE
A number of SC notes only apply to non-web software or only to non-web documents but not noted that way

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#692]
See WCAG2ICT's section [Applying SC 2.1.1 Keyboard to non-web documents and software](https://w3c.github.io/wcag2ict/#applying-sc-2-1-1-keyboard-to-non-web-documents-and-software). There are three notes, but all seem to only apply to non-web software. The EN 301 549 seemed to agree with that because none of the notes were added to clause 10.2.1.1 Keyboard. 

Just want to call attention to this, and if we agree the WCAG2ICT added notes are only for non-web software we need to add the parenthetic statement "(for non-web software)"  to the WCAG2ICT section right before the notes to make it clear.

There are other instances where we didn't specify the parenthetic as well but maybe should have:
- [x] 1.3.1 Info and Relationships - has a note that clearly only applies to non-web software - add parenthetic "(for non-web software)" on the line before the note.
- [x] 1.4.12 Text Spacing - has a note that only applies to non-web software and not identified as such. (Notes 2 and 4 are software-only, so Note 2 gets moved down and becomes Note 3
- [x] 2.1.1 Keyboard - 3 notes were not added for EN 301 549 10.2.1.1 (See also Issue #691 where EN also changed the notes.) All 3 notes are software-only
- [x] 2.1.2 No Keyboard Trap - Note 3 appears to only apply to non-web software (EN didn't include Note 3 in clause 10.2.1.2) Notes 3 and 4 are software-only
- [x] 2.1.4 Character Key Shortcuts - Note 1 appears to only apply to non-web software (EN didn't include the note in clause 10.2.1.4.) Both notes are software-only
- [x] 2.4.4 Link Purpose (in context) - Note should say "In non-web software" Both notes are software-only
- [x] 2.4.6 Headings and Labels - Note should say "In non-web software" - it's software-only
- [x] 3.1.1 Language of Page - added notes only apply to non-web software. Both notes are software-only
- [x] 3.3.8 Accessible Authentication - Notes 3 and 4 seem to only apply to non-web software yet the EN 301 549 applies both notes non-web documents and to non-web software. (MJM: I think they're really just for non-web software.) Notes 3, 4, and 5 are non-web software
- [x] 4.1.2 Name, Role, Value - Note 2 seems to only apply to non-web software, and Note 3 seems to only apply to non-web documents. If that is the case, the EN 301 549 needs to remove Notes to match. 1, 2, and 4 are software-only and note 3 is documents-only. This will cause a reordering where the note that starts with "For non-web document formats..." is the only note listed for non-web documents, and the rest are for non-web software.

**Comment by maryjom** [comment]
I opened #713 to track the additional instances I found. Will wait to make further changes as we may end up marking these differently on each individual note rather than using a parenthetic before a group of notes which is more accessible. Re-closing this issue as the original problem has been clarified/fixed in WCAG2ICT.
